### ISSUE TITLE
How do we handle language translations

### ISSUE THREAD

**Issue by zvonimirfras** [carbon-design-system/carbon-charts#260]
We need a way to let users translate hardcoded strings (like the ones on aria-labels).

In carbon-components-angular, we have (simply put) inputs that initialize from json files and users can provide either json files or override inputs directly, it's been working great so far.

Something that provides similar functionality would be awesome here.

Discuss! :)

**Comment by cal-smith** [comment]
I think short term we can keep it in the config ... long term we'll probably want to look at integrating something from i18next https://www.i18next.com/

Especially considering we'll need to provide support for things like numbers/time/date/currency/etc

**Comment by cal-smith** [comment]
Something I'm thinking on as I tinker with the refactor is how we want to support theming ... I18n will follow the same pattern as the theme provider I think

**Comment by theiliad** [comment]
Yes, definitely. I like the idea of providers. Theme and i18n should also work in a similar way

**Comment by douglasward** [comment]
What's the status here? We really need to provide translations for our product and also handle the localization of numbers/dates/currencies etc.

**Comment by nstuyvesant** [comment]
@zvonimirfras - does this issue (https://github.com/carbon-design-system/carbon-charts/issues/1635) address what you're running into? If so, perhaps consider closing this one so we can consolidate. Feel free to update the other issue if you'd like to see anything changed.
